tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -566,18 +535,18 @@ def test_work_order_workerEncryptionKey_special_character(self):
         result_response = self.test_obj.getresult(
             self.test_obj.build_request_output['request_obj'])
 
-        # assert (
-        #         verify_test(
-        #             result_response, 0,
-        #             self.test_obj.build_request_output['pre_test_output'],
-        #             self.test_obj.build_request_output['action_obj'])
-        #         is TestStep.SUCCESS.value)
-
         assert (
```

Comment:
Similar to above comment, the assert got changed from check_negative to verify_test.

Reply:
Reverting back to check_negative_test as i have to revisit the test to get confirmation from Ram on the behavior.

Reply:
changes addressed in commit "3010f9e43d2d6a3cea2f19a464bd6efbae48e964"